test_scripts/PROJECTION/000_Start_secured_audio_service.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@dboltovskyi , 001 and 002 are simple scripts, so combined in one according to comment.
But 003 and 004 are still separated, because scripts are related to streaming and seconds step(start audio streaming) in script can be affected by first(video streaming).